theme/themes/eea/globals/reset.overrides

```diff
@@ -193,6 +193,15 @@ sup {
   top: -0.5em;
 }
 
+/**
+ * Add background and color for scroll-to-text fragments
+ */
+
+::target-text {
```

Comment:
@kreafox  add the values please as variables in site.variables and move the styles to site.overrides

Reply:
@ichim-david Done